04_setup_ironic.sh

```diff
@@ -49,6 +49,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
     echo "RUN sed -i 's%$OLDIMAGE%${!IMAGE_VAR}%g' /release-manifests/*" >> $DOCKERFILE
 done
 
+if [ ! -z "${MIRROR_IMAGES}" ]; then
+    oc adm release mirror \
+       --insecure=true \
+        -a pull-secret.json  \
```

Comment:
There's an example in ocp_install_env.sh of how we can write the pull secret to a temporary file, which should get cleaned up later:

https://github.com/openshift-metal3/dev-scripts/blob/06516bd606e8eac5942602b17c12cc641b0363d4/ocp_install_env.sh#L21-L24

Reply:
Ugh, yes, I saw that! Somehow that change didn't make it back from the server to my working copy.

Reply:
Done